flang/lib/Optimizer/CodeGen/CodeGen.cpp

```diff
@@ -95,6 +95,20 @@ genConstantIndex(mlir::Location loc, mlir::Type ity,
   return rewriter.create<mlir::LLVM::ConstantOp>(loc, ity, cattr);
 }
 
+template <typename OpType>
+static OpType getSingleOpOfType(Block &block) {
```

Comment:
This is imported from `mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp`. I'm checking with MLIR to move it to a central place where it can be used by anyone (it's already duplicated in MLIR).

Reply:
Phabricator patch is up for review: https://reviews.llvm.org/D110145

Reply:
This is not needed anymore.